docs/admin/performance_tuning.md

```diff
@@ -398,11 +334,153 @@ Completed test=FETCH
     environment variable can be set to 1 to take advantage of the extended
     asynchronous DRAM refresh (eADR) feature
 
-In DAOS mode, daos\_perf can be used as an MPI application like IOR.
-Parameters are the same, except that `-T daos` can be used to select the daos
-mode. This option can be omitted too since this is the default.
+A tool called daos\_perf with the same syntax as vos\_perf is also available
+to run tests from a compute node with the full DAOS stack. Please refer
+to the next section for more information.
+
+### SSDs
+
+Performance of SSDs can be measured directly with SPDK via the spdk_nvme_perf
+tool. It can be run to test bandwidth in a non-destructive way as follows:
+
+```bash
+spdk_nvme_perf -q 16 -o 1048576 -w read -c 0xff -t 60
+```
+
+IOPS can be measured with the following command:
+```bash
+spdk_nvme_perf -q 16 -o 4096 -w read -c 0xff -t 60
+```
+
+`-q` is used to control the queue depth, `-o` for the I/O size, `-w` is the
+operation and can be either (rand)read, (rand)write or (rand)rw. The test
```

Comment:
!!!note : *write and *rw are destructive

Reply:
fixed.